src/main/java/com/taskadapter/redmineapi/AttachmentManager.java

```diff
@@ -110,7 +110,41 @@ public Attachment uploadAttachment(String contentType, File content)
         }
     }
 
+/**
+	 * Uploads an attachment.
+	 * 
+	 * @param fileName
+	 *            file name of the attachment.
+	 * @param contentType
+	 *            content type of the attachment.
+	 * @param content
+	 *            attachment content stream.
+	 * @return attachment content.
+	 * @throws RedmineException
+	 *             if something goes wrong.
+	 * @throws IOException
+	 *             if input cannot be read. This exception cannot be thrown yet (I am not sure if http client can distinguish "network" errors and local errors) but is will be good to distinguish
+	 *             reading errors and transport errors.
+	 */
+	public Attachment uploadAttachment(File file, String contentType) throws RedmineException, IOException {
+		final String token;
+		try {
+			token = transport.upload(file);
+			final Attachment result = AttachmentFactory.create();
+			result.setToken(token);
+			result.setContentType(contentType);
```

Comment:
What if I call this method like
uploadAttachment(new File("image.png"), "text/plain") ?
This method would return result with content type "text/plain". But what would I get when I download the same attachement from the server using redmine java api? Would it be "text/plain" (as asked and returned from this call) or "image/png" as deducted from the file extension?